scripts/ci/build_push_images.sh

```diff
@@ -38,14 +38,27 @@ docker build -t ${node_tag_specific} $taglatestflag -f Dockerfile-csi-node --bui
 docker push ${node_tag_specific}
 [ "$tag_latest" = "true" ] && docker push ${node_tag_latest} || :
 
+# Host Definer
```

Comment:
it looks very similar to node and controller, is there a way to have that logic once and call it with different parameters?

Reply:
I think it's a problem that we had before this change, can I open a ticket to fix it next sprint?

Reply:
could we finish at least the second part of CSI-1883 in this sprint?

Reply:
@ArbelNathan CSI-4528

Reply:
@oriyarde I am not sure if it relates to this PR, but yes I can try and do it, anyway you can mention it in the daily, that Eran will prioritize it over other tickets